_board/stackrduino_m0_pro.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: download
+board_id: "stackrduino_m0_pro"
+title: "StackRduino M0+ PRO Download"
+name: "StackRduino M0+ PRO"
```

Comment:
You deleted the manufacturer from some other titles and names, so I'm doing that here.

```suggestion
title: "M0+ PRO Download"
name: "M0+ PRO"
```

Reply:
I left this one because the official board name has the manufacturer name in it.

Reply:
:+1: